examples/ember-htmlbars/README.md

```diff
@@ -1,12 +1,12 @@
-# Compile templates for use in Ember.js and HTMLBars
+# Compile HTMLBars templates for Ember.js
 
-This example will show you how to compile handlebars#2.0.0 templates for use in Ember.js applications with HTMLBars.
+This example will show you how to compile HtmlBars templates for use in Ember.js applications.
 
 ## Dependencies
 
-* [`ember-handlebars`](https://www.npmjs.org/package/ember-handlebars) - Compile templates for Ember.js
 * [`gulp-concat`](https://www.npmjs.org/package/gulp-concat) - Combine output into a single file
-* [`ember-cli-htmlbars`](https://www.npmjs.org/package/ember-cli-htmlbars) - Compile templates for Ember.js with HTMLBars
+* [`gulp-replace`](https://www.npmjs.com/package/gulp-replace) - A string replace plugin for gulp
+* [`gulp-wrap-amd`](https://github.com/phated/gulp-wrap-amd) - Transforming templates to AMD style used in browsers.
```

Comment:
Do we think that `gulp-wrap-amd` is best included in a separate example instead of the base ember-htmlbars example? If everyone in Ember land is using AMD, I'm all for it, but I'd rather do the bare minimum to avoid confusing folks.